src/rgw/setup/rgw.py

```diff
@@ -117,6 +117,17 @@ def config(conf: MappedConf):
         os.environ['M0_TRACE_DIR'] = motr_trace_dir
         Log.info('Created motr trace directory : %s' % os.environ.get('M0_TRACE_DIR'))
 
+        # Create motr addb directory for collecting addb files
+        # in case admin user creation issue during mini-provisioner execution.
+        Log.info('Creating motr addb_files- directory for collecting addb files..')
+        motr_fid_key = const.MOTR_ADMIN_FID_KEY
+        motr_fid_value = Conf.get(Rgw._conf_idx, motr_fid_key)
+        log_path = Rgw._get_log_dir_path(conf)
+        addb_dir = os.path.join(log_path, f'addb_files-{motr_fid_value}')
+        os.makedirs(addb_dir, exist_ok=True)
+        os.environ['M0_CLIENT_ADDB_DIR'] = addb_dir
```

Comment:
why set twice, its also set in src/rgw/setup/rgw_service.py @Line 34

Reply:
In init container as well as in runtime container, we are setting this environment variable.
hence we added it in config phase and in start logic.

Reply:
oh ok, these are not in same flow